background.js

```diff
@@ -101,8 +109,7 @@ function bufferData(tabId, message) {
 // context script –> background
 chrome.runtime.onMessage.addListener(brokerMessage);
 
-chrome.runtime.onConnect.addListener(function(devToolsPort) {
-
+chrome.runtime.onConnect.addListener(function (devToolsPort) {
```

Comment:
It falls under the random changes category :smiley:
Both variations are used throughout the project - it drives me crazy. I was trying to blend in with the rest of the file :grin: